src/store/actions.ts

```diff
@@ -25,6 +25,15 @@ export default ( searchEntityRepository: SearchEntityRepository, metricsCollecto
 			return searchResult;
 		} );
 	},
+	async searchItemValues(
+		_context: ActionContext<RootState, RootState>,
+		options: SearchOptions ): Promise<SearchResult[]> {
+		// check for empty
+		const searchResults = await searchEntityRepository.searchItemValues( options.search, 12, options.offset );
```

Comment:
i don't know where this 12 comes from, i took it from `searchProperties`, does it make sense?

Reply:
That sounds like a very good idea!